node/hubsConfig.js

```diff
@@ -0,0 +1,13 @@
+
+
+var hubs = [
```

Comment:
kk do we need a file header here?

---
In reply to: [89002812](https://github.com/openT2T/cloud/pull/9#discussion_r89002812) [](ancestors = 89002812,88997289)